### ISSUE TITLE
Error playing Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv

### ISSUE THREAD

**Issue by ckrzen** [keredson/gnomecast#98]
Loads into app, but does not transcode or seem to do anything.

------------------------------------------------------------

File: Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv
Video: Video #1 (h264)
Audio: eng (opus,)
Subtitles: 
Device: Chromecast (Unknown manufacturer)
Chromecast: v1.9.2

FileMetadata(fn:/home/chris/Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv, ready:False, thumbnail_fn:/tmp/gnomecast_thumbnail_9iydbgfc.jpg, container:mkv, video_streams:[StreamMetadata(index:0:0, codec:h264, title:Video #1)], audio_streams:[StreamMetadata(index:0:1, codec:opus,, title:eng, mono:False)], subtitles:[])

```ffmpeg version 4.1.4 Copyright (c) 2000-2019 the FFmpeg developers
Stream #0:0: Video: h264 (High), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 1k tbn, 59.94 tbc (default)
Stream #0:1(eng): Audio: opus, 48000 Hz, stereo, fltp (default)```

**Comment by ckrzen** [comment]
OK. I think I've been able to produce some debug for you:

`$ gnomecast 
update_button_visible
None
update_button_visible
update_button_visible
serving on http://192.168.1.72:39059
check_ffmpeg
b'/usr/bin/ffmpeg\n'
Chromecast('192.168.1.70', port=8009, device=DeviceStatus(friendly_name='Poodle Lounge TV', model_name='Chromecast', manufacturer='Unknown manufacturer', uuid=UUID('29466b4d-975e-ea46-7fdd-9c91dc866bff'), cast_type='cast'))
update_button_visible
update_button_visible
update_button_visible
Open clicked
File selected: ['/home/chris/Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv']
update_button_visible
STOP command requested but no session is active.
STOP command requested but no session is active.
update_button_visible
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib64/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/chris/.local/lib/python3.7/site-packages/gnomecast.py", line 1060, in update_transcoders
    if not self.video_stream: self.video_stream = fmd.video_streams[0]
AttributeError: 'FileMetadata' object has no attribute 'video_streams'

update_button_visible
update_button_visible
video: Stream #0:0: Video: h264 (High), yuv420p(tv, bt709, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 1k tbn, 59.94 tbc (default) '0:0'
audio: Stream #0:1(eng): Audio: opus, 48000 Hz, stereo, fltp (default)
['ffmpeg', '-y', '-i', '/home/chris/Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv', '-vn', '-an']
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib64/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/chris/.local/lib/python3.7/site-packages/gnomecast.py", line 175, in parse
    self.load_subtitles()
  File "/home/chris/.local/lib/python3.7/site-packages/gnomecast.py", line 189, in load_subtitles
    output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib64/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/usr/lib64/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['ffmpeg', '-y', '-i', '/home/chris/Joel Edgerton to Return as Owen Lars in Obi-Wan Series - Jedi Council-pCU2GE07KW4.mkv', '-vn', '-an']' returned non-zero exit status 1.
`

**Comment by keredson** [comment]
nice!  you're the first person to use the new "report file" button!

**Comment by keredson** [comment]
give v1.9.3 a whirl see if that's better.  (`pip3 install --upgrade gnomecast`)

**Comment by ckrzen** [comment]
Thanks! That fixed the transcoding and video plays, just _no_ sound,

`$ pip3 install --upgrade --user youtube_dl gnomecast
Requirement already up-to-date: youtube_dl in ./.local/lib/python3.7/site-packages (2019.9.28)
Collecting gnomecast
  Downloading https://files.pythonhosted.org/packages/2d/b6/5f9a3189e377d657d63fe5ad72361d08d6850db02779ae45fa63c83037fb/gnomecast-1.9.3.tar.gz
Requirement already satisfied, skipping upgrade: pychromecast in ./.local/lib/python3.7/site-packages (from gnomecast) (3.2.2)
Requirement already satisfied, skipping upgrade: bottle in ./.local/lib/python3.7/site-packages (from gnomecast) (0.12.17)
Requirement already satisfied, skipping upgrade: pycaption in ./.local/lib/python3.7/site-packages (from gnomecast) (1.0.1)
Requirement already satisfied, skipping upgrade: paste in /usr/lib/python3.7/site-packages (from gnomecast) (2.0.3)
Requirement already satisfied, skipping upgrade: html5lib in ./.local/lib/python3.7/site-packages (from gnomecast) (1.0.1)
Requirement already satisfied, skipping upgrade: zeroconf>=0.17.7 in ./.local/lib/python3.7/site-packages (from pychromecast->gnomecast) (0.23.0)
Requirement already satisfied, skipping upgrade: requests>=2.0 in /usr/lib/python3.7/site-packages (from pychromecast->gnomecast) (2.21.0)
Requirement already satisfied, skipping upgrade: protobuf>=3.0.0 in ./.local/lib/python3.7/site-packages (from pychromecast->gnomecast) (3.9.0)
Requirement already satisfied, skipping upgrade: casttube>=0.2.0 in ./.local/lib/python3.7/site-packages (from pychromecast->gnomecast) (0.2.0)
Requirement already satisfied, skipping upgrade: beautifulsoup4<4.5.0,>=4.2.1 in ./.local/lib/python3.7/site-packages (from pycaption->gnomecast) (4.4.1)
Requirement already satisfied, skipping upgrade: lxml>=3.2.3 in ./.local/lib/python3.7/site-packages (from pycaption->gnomecast) (4.4.0)
Requirement already satisfied, skipping upgrade: cssutils>=0.9.10 in ./.local/lib/python3.7/site-packages (from pycaption->gnomecast) (1.0.2)
Requirement already satisfied, skipping upgrade: future in ./.local/lib/python3.7/site-packages (from pycaption->gnomecast) (0.17.1)
Requirement already satisfied, skipping upgrade: enum34 in ./.local/lib/python3.7/site-packages (from pycaption->gnomecast) (1.1.6)
Requirement already satisfied, skipping upgrade: six>=1.9.0 in /usr/lib/python3.7/site-packages (from pycaption->gnomecast) (1.12.0)
Requirement already satisfied, skipping upgrade: webencodings in ./.local/lib/python3.7/site-packages (from html5lib->gnomecast) (0.5.1)
Requirement already satisfied, skipping upgrade: ifaddr in ./.local/lib/python3.7/site-packages (from zeroconf>=0.17.7->pychromecast->gnomecast) (0.1.6)
Requirement already satisfied, skipping upgrade: chardet<3.1.0,>=3.0.2 in /usr/lib/python3.7/site-packages (from requests>=2.0->pychromecast->gnomecast) (3.0.4)
Requirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in /usr/lib/python3.7/site-packages (from requests>=2.0->pychromecast->gnomecast) (2.7)
Requirement already satisfied, skipping upgrade: urllib3<1.25,>=1.21.1 in /usr/lib/python3.7/site-packages (from requests>=2.0->pychromecast->gnomecast) (1.24.3)
Requirement already satisfied, skipping upgrade: setuptools in /usr/local/lib/python3.7/site-packages (from protobuf>=3.0.0->pychromecast->gnomecast) (41.2.0)
Installing collected packages: gnomecast
  Running setup.py install for gnomecast ... done
Successfully installed gnomecast-1.9.3`

**Comment by keredson** [comment]
sound for chromecast (non-ultra), home, home mini fixed in v1.9.4.
